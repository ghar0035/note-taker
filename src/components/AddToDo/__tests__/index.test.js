import renderer from "react-test-renderer";
import AddTodo from "../index.js";
import { MemoryRouter as Router } from "react-router-dom";

test("renders add todo", () => {
    const tree = renderer
      .create(
        <Router>
          <AddTodo />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });