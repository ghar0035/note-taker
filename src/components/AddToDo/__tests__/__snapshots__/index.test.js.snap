// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders add todo 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="sc-bcXHqe gNwiIb"
  >
    <div
      className="form-control"
    >
      <label
        htmlFor="title"
      >
        Title
      </label>
      <input
        id="title"
        name="title"
        onChange={[Function]}
        required={true}
        type="text"
        value=""
      />
    </div>
    <div
      className="form-control"
    >
      <label
        htmlFor="due_date"
      >
        Due date
      </label>
      <input
        id="due_date"
        name="due_date"
        onChange={[Function]}
        required={true}
        type="date"
        value=""
      />
    </div>
    <div
      className="form-control"
    >
      <label
        htmlFor="content"
      >
        Content
      </label>
      <textarea
        id="content"
        name="content"
        onChange={[Function]}
        required={true}
        value=""
      />
    </div>
    <div
      className="btn-group"
    >
      <button
        className="btn secondary"
        onClick={[Function]}
        type="button"
      >
        Cancel
      </button>
      <button
        className="btn"
        type="submit"
      >
        Add to do
      </button>
    </div>
  </div>
</form>
`;
